tsconfig.json

```diff
@@ -11,6 +11,7 @@
         "downlevelIteration": true,
         "baseUrl": ".",
         "esModuleInterop": true,
+        "skipLibCheck": true,
```

Comment:
What is the reason for adding this?

Reply:
```
$ tsc
node_modules/@types/node/globals.d.ts:72:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; abort(reason?: any): AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; timeout(milliseconds: number): AbortSignal; }'.

72 declare var AbortSignal: {
               ~~~~~~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:2090:13
    2090 declare var AbortSignal: {
                     ~~~~~~~~~~~
    'AbortSignal' was also declared here.

../crypto-wasm/lib/type_declarations/index.d.ts:3:1 - error TS2308: Module "./bbs_plus" has already exported a member named 'DEFAULT_BLS12381_G1_PUBLIC_KEY_LENGTH'. Consider explicitly re-exporting to resolve the ambiguity.

3 export * from "./bbs";
  ~~~~~~~~~~~~~~~~~~~~~~

../crypto-wasm/lib/type_declarations/index.d.ts:3:1 - error TS2308: Module "./bbs_plus" has already exported a member named 'DEFAULT_BLS12381_G2_PUBLIC_KEY_LENGTH'. Consider explicitly re-exporting to resolve the ambiguity.

3 export * from "./bbs";
  ~~~~~~~~~~~~~~~~~~~~~~

../crypto-wasm/lib/type_declarations/index.d.ts:3:1 - error TS2308: Module "./bbs_plus" has already exported a member named 'DEFAULT_BLS12381_PRIVATE_KEY_LENGTH'. Consider explicitly re-exporting to resolve the ambiguity.

3 export * from "./bbs";
  ~~~~~~~~~~~~~~~~~~~~~~


Found 4 errors in 2 files.

Errors  Files
     1  node_modules/@types/node/globals.d.ts:72
     3  ../crypto-wasm/lib/type_declarations/index.d.ts:3
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
I'll update exported constants in `crypto-wasm-ts` soon

Reply:
https://github.com/docknetwork/crypto-wasm/pull/8